### PR TITLE
Remove unnecessary `/usr/sbin` and `/sbin` references

### PR DESCRIPTION
**Pull request by kidonng**
See #1688

A [remaining reference](https://github.com/matryer/xbar-plugins/blob/b74b4799e6020df9e2d88c6ef536e852e167840c/Network/updatevpnbookpass.1d.sh#L73) is in `Network/updatevpnbookpass.1d.sh` as it also references a path `/Applications/Utilities/Keychain Access.app` which no longer existing on newer systems, so I guess that line is broken (hence commented).

### CODE REVIEW

File: Dev/Kubernetes/minikube.5s.sh [hunk 1]
```diff
@@ -13,7 +13,6 @@ MINIKUBE_LOGO_BW='iVBORw0KGgoAAAANSUhEUgAAAIAAAAB8CAYAAAChbripAAAABGdBTUEAALGPC/
 
 LANG="en_US.UTF-8"
 # You may need to update the PATH to include minikube and kubectl
-#PATH="/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
```
Comment (kidonng) [inline Dev/Kubernetes/minikube.5s.sh:16]: This line only prepends `/usr/local/sbin` to the default path but `minikube` or `kubectl` won't   be there, so this line essentially does nothing.

**Comment by hagemt** [comment]
+1 for this; I noticed this is a problem in all three Homebrew dev plugins, too.

File: Dev/Kubernetes/minikube.5s.sh (hunk 1 above)
Comment (kdeldycke) [reply Dev/Kubernetes/minikube.5s.sh]: For reference, this is the original commit for `meta_package_manager.7h.py`: https://github.com/kdeldycke/meta-package-manager/commit/f35abda586d4275f730c5a9e74805a8a1eb6d35a

**Comment by matryer** [comment]
thanks @kidonng

**Pull request merged**